src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/service/UserService.kt

```diff
@@ -60,6 +61,16 @@ class UserService(
     } ?: throw UsernameNotFoundException("User $userId not found")
   }
 
+  @Transactional
+  @Throws(UserGroupRelationshipException::class, UsernameNotFoundException::class)
```

Comment:
Did we say we're leaving off the "throws" ?

Reply:
UsernameNotFoundException is runtime so have removed that, the other is a checked exception which is actually thrown so think that needs to remain in this case.